Adyen/util.py

```diff
@@ -45,7 +47,8 @@ def is_valid_hmac_notification(dict_object, hmac_key):
             raise ValueError("Must Provide hmacSignature in additionalData")
         else:
             expected_sign = dict_object['additionalData']['hmacSignature']
-            del dict_object['additionalData']
+            if 'additionalData' in dict_object:
```

Comment:
is this different from the check on line 45?

Reply:
Yeah indeed is the same. Check again ? @Aleffio @wboereboom